--- a/browser/branding/floorp-official/configure.sh
+++ b/browser/branding/floorp-official/configure.sh
@@ -5,5 +5,4 @@
 MOZ_APP_DISPLAYNAME="Floorp"
 MOZ_MACBUNDLE_ID=Floorp
 MOZ_APP_NAME="floorp"
-MOZ_APP_VENDOR="Ablaze"
 MOZ_APP_REMOTINGNAME="floorp"
--- a/browser/moz.configure
+++ b/browser/moz.configure
@@ -11,7 +11,7 @@
 imply_option("MOZ_PROFILE_MIGRATOR", True)
 
 
-imply_option("MOZ_APP_VENDOR", "Mozilla")
+imply_option("MOZ_APP_VENDOR", "Ablaze")
 imply_option("MOZ_APP_ID", "{ec8030f7-c20a-464f-9b0e-13a3a9e97384}")
 # Include the DevTools client, not just the server (which is the default)
 imply_option("MOZ_DEVTOOLS", "all")
